lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -14,9 +14,17 @@ namespace lldb_private {
 bool LLDBMemoryReader::queryDataLayout(DataLayoutQueryType type, void *inBuffer,
                                        void *outBuffer) {
   switch (type) {
-  // FIXME: add support for case DLQ_GetPtrAuthMask rdar://70729149
-  case DLQ_GetPtrAuthMask:
-    return false;
+  case DLQ_GetPtrAuthMask: {
+    assert(m_process.GetCodeAddressMask() == m_process.GetDataAddressMask() &&
+           "not supported");
```

Comment:
Nit: different code and address masks are not supported?

Reply:
Does this code know if the mask returned is used for data or for code addresses?  Linux standardized on the code/data masks, but I don't think the distinction is possible in AArch64, I'm not sure what target motivated that choice.  I think a sanity check of ensuring that they're the same, if we don't know what this mask might be used for, is correct.

Reply:
The swift::reflection::MemoryReader API doesn't distinguish between the two.